price/priceService.ts

```diff
@@ -0,0 +1,21 @@
+const baseUrl:string = 'https://dydx-testnet.imperator.co/v4';
+
+function fetchCandles(pair:string,timeFrame:string):void{
```

Comment:
1. This async function should return the candles. Think about it: How plugins are meant to use the `data` that you logged?
2. Instead of `string`, use the `Pair` type alias from `common.ts`. While they are exactly the same, it improves readability. You can also do the same for `timeframe`.
3.